File: July-LeetCoding-Challenge/14-Angle-Between-Hands-of-a-Clock/Angle-Between-Hands-of-a-Clock.cpp
class Solution {
public:
    double angleClock(int hour, int minutes) {
        double f1 = (hour % 12) * 30.0 + 0.5 * minutes;
        double f2 = 6.0 * minutes;
        double t = abs(f1 - f2);
        return min(t, 360 - t);
    }
};	

File: July-LeetCoding-Challenge/14-Angle-Between-Hands-of-a-Clock/Angle-Between-Hands-of-a-Clock.cs
public class Solution {
    public double AngleClock(int hour, int minutes) {
        var angle = Math.Abs((double)(hour + (double)minutes/60) / 12 * 360 - (double)minutes / 60 * 360);
        if(angle >= 180) angle = (double)360 - angle;
        return Math.Round(angle, 1);
    }
}